packages/shared/components/Select/Select.story.tsx

```diff
@@ -0,0 +1,88 @@
+/*
```

Comment:
since we have this file to showcase light vs dark, lets create a normal searchable selector too (not multis)